auth/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = function (app, hexo) {
       if (req.method === 'POST') {
           req.authenticate(['adminAuth'], function(error, done) {
               if (done) {
-                  res.writeHead(302, { 'Location':  "/admin/" });
+                  res.writeHead(302, { 'Location':  hexo.config.root + "admin/" });
```

Comment:
i try to print the hexo.config.root, it contain two slash.
if my root is blog, the hexo.config.root is "/blog/".
so i update strategy.js to remove slash before admin.

thanks